test/Microsoft.ML.Tests/Transformers/LineParserTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.ML.Runtime.Internal.Utilities;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.ML.Tests.Transformers
+{
+    public class LineParserTests
+    {
+        public static IEnumerable<object[]> ValidInputs()
+        {
+            foreach (var line in new string[]
+            {
+                "key 0.1 0.2 0.3", "key 0.1 0.2 0.3 ",
```

Comment:
I would make this 4 lines instead of 2. The way it reads is that there are 2 lines, but you have to parse the lines to see that there are really 4.